php/class-wp-js-widget.php

```diff
@@ -100,7 +102,7 @@ public function enqueue_frontend_scripts() {}
 	 * @return array Schema.
 	 */
 	public function get_item_schema() {
-		$schema = array(
+		$item_schema = array(
 			'title' => array(
 				'description' => __( 'The title for the widget.', 'js-widgets' ),
 				'type' => array( 'string', 'object' ),
```

Comment:
These compound types are primarily useful for the REST API. In responses, they allow for the `raw`/`rendered` pairs.